frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -113,22 +150,70 @@ export class ResourceMenuBase extends React.Component<Props, State> {
                         autoFocus
                         value={ this.state.search }
                         onChange={ this.updateResourceList }
+                        placeholder="Filter resources"
+                    />
+                </div>
+
+                <div className="resources-list-header">
+                    <Localized id="resource-ResourceMenu--resource">
+                        <span className="resource"
+                            onClick={ this.sortByResource }>Resource</span>
+                    </Localized>
+                    <span className={"resource icon " + resourceClass}
+                        onClick={ this.sortByResource }
+                    />
+                    <Localized id="resource-ResourceMenu--progress">
+                        <span className="progress"
+                            onClick={ this.sortByProgress }>Progress</span>
+                    </Localized>
+                    <span className={"progress icon " + progressClass}
+                        onClick={ this.sortByProgress }
                     />
                 </div>
 
                 <ul>
                     { resourceElements.length ?
-                        resourceElements.map((resource, index) => {
-                            return <ResourceItem
-                                parameters={ parameters }
-                                resource={ resource }
-                                navigateToPath={ this.navigateToPath }
-                                key={ index }
-                            />;
-                        })
+                        (this.state.sortActive === 'resource' ?
+                            resourceElements.sort((a, b) => {
+                                const resourceA = this.getResource(a);
+                                const resourceB = this.getResource(b);
+
+                                let result = 0;
+
+                                if (resourceA < resourceB) {
+                                    result = -1;
+                                }
+                                if (resourceA > resourceB) {
+                                    result = 1;
+                                }
+                                return this.state.sortAsc ? result :  result * -1;
```

Comment:
Remove double space after `:`. And add a blank line above this one. :)

Also below.